src/libs/PaymentUtils.js

```diff
@@ -0,0 +1,72 @@
+import _ from 'underscore';
+import lodashGet from 'lodash/get';
+import * as Expensicons from '../components/Icon/Expensicons';
+import getBankIcon from '../components/Icon/BankIcons';
+import CONST from '../CONST';
+import * as Localize from './Localize';
+
+/**
+ * Get the PaymentMethods list
+ * @param {Array} bankAccountList
+ * @param {Array} cardList
+ * @param {String} [payPalMeUsername='']
+ * @returns {Array<PaymentMethod>}
+ */
+function getPaymentMethods(bankAccountList, cardList, payPalMeUsername = '') {
+    const combinedPaymentMethods = [];
+
+    _.each(bankAccountList, (bankAccount) => {
+        // Add all bank accounts besides the wallet
+        if (bankAccount.type === CONST.BANK_ACCOUNT_TYPES.WALLET) {
+            return;
+        }
+
+        const formattedBankAccountNumber = bankAccount.accountNumber
+            ? `${Localize.translateLocal('paymentMethodList.accountLastFour')} ${bankAccount.accountNumber.slice(-4)
+            }`
+            : null;
+        const {icon, iconSize} = getBankIcon(lodashGet(bankAccount, 'additionalData.bankName', ''));
+        combinedPaymentMethods.push({
+            type: CONST.PAYMENT_METHODS.BANK_ACCOUNT,
+            title: bankAccount.addressName,
+            description: formattedBankAccountNumber,
+            methodID: bankAccount.bankAccountID,
+            icon,
+            iconSize,
+            key: `bankAccount-${bankAccount.bankAccountID}`,
+        });
+    });
+
+    _.each(cardList, (card) => {
+        const formattedCardNumber = card.cardNumber
+            ? `${Localize.translateLocal('paymentMethodList.cardLastFour')} ${card.cardNumber.slice(-4)}`
+            : null;
+        const {icon, iconSize} = getBankIcon(card.bank, true);
+        combinedPaymentMethods.push({
+            type: CONST.PAYMENT_METHODS.DEBIT_CARD,
+            title: card.addressName,
+            description: formattedCardNumber,
+            methodID: card.cardNumber,
+            icon,
+            iconSize,
+            key: `card-${card.cardNumber}`,
+        });
+    });
+
+    if (payPalMeUsername) {
+        combinedPaymentMethods.push({
+            type: CONST.PAYMENT_METHODS.PAYPAL,
+            title: 'PayPal.me',
+            methodID: CONST.PAYMENT_METHODS.PAYPAL,
```

Comment:
Remove

Reply:
This is used for the onPress handler. onPress handler expects different values for different types of payment methods. Should I directly add onPress handler here?

Reply:
>Should I directly add onPress handler here?

No then you'd need to pass a function to this method which seems weird

The change is fine I was just recommending that we do not add this in this PR because it's hard to see why the change was made without carefully reviewing the code. But now that I'm looking at this - I'm not sure how else you could do it.